central/vulnmgmt/service/service_impl.go

```diff
@@ -0,0 +1,91 @@
+package service
+
+import (
+	"context"
+	"time"
+
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/pkg/errors"
+	deploymentDS "github.com/stackrox/rox/central/deployment/datastore"
+	imageDS "github.com/stackrox/rox/central/image/datastore"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/auth/permissions"
+	"github.com/stackrox/rox/pkg/errox"
+	"github.com/stackrox/rox/pkg/grpc/authz"
+	"github.com/stackrox/rox/pkg/grpc/authz/perrpc"
+	"github.com/stackrox/rox/pkg/grpc/authz/user"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	"github.com/stackrox/rox/pkg/search"
+	"google.golang.org/grpc"
+)
+
+var (
+	authorizer = perrpc.FromMap(map[authz.Authorizer][]string{
+		user.With(permissions.View(resources.Deployment), permissions.View(resources.Image)): {
+			"/v1.VulnMgmtService/VulnMgmtExportWorkloads",
+		},
+	})
+	log = logging.LoggerForModule()
+)
+
+// serviceImpl provides APIs for workload vulnerabilities.
+type serviceImpl struct {
+	v1.UnimplementedVulnMgmtServiceServer
+
+	deployments deploymentDS.DataStore
+	images      imageDS.DataStore
+}
+
+// RegisterServiceServer registers this service with the given gRPC Server.
+func (s *serviceImpl) RegisterServiceServer(grpcServer *grpc.Server) {
+	v1.RegisterVulnMgmtServiceServer(grpcServer, s)
+}
+
+// RegisterServiceHandler registers this service with the given gRPC Gateway endpoint.
+func (s *serviceImpl) RegisterServiceHandler(ctx context.Context, mux *runtime.ServeMux, conn *grpc.ClientConn) error {
+	return v1.RegisterVulnMgmtServiceHandler(ctx, mux, conn)
+}
+
+// AuthFuncOverride specifies the auth criteria for this service.
```

Comment:
super-nit (consistency): The other services I checked have a slightly different wording.
```suggestion
// AuthFuncOverride specifies the auth criteria for this API.
```

Reply:
ack. Then let's leave it in the newer form.